### DISCUSSION TITLE
Freeplane community online conference on Saturday, May 28, 2022

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#339] [Announcements]
Join us for a roundtable to talk about anything related to Freeplane. Your curiosity sets up the agenda. You can ask any questions or present any idea or experience in a short lightning talk. The Lean Coffee meeting format allows us to be flexible and to talk about everything that most interests you as long as needed.

A very quick video introduction to Lean Coffee is available at https://www.youtube.com/watch?v=4TOOdAXwcrk .

We meet at an exclusive online location https://spatial.chat/s/freeplane.
The online service allows only 25 people to join, and I hope it is the right number.

Time: Saturday, May 28, 2022
13:00 London
14:00 Amsterdam, Berlin, Rome, Stockholm, Vienna
online, https://spatial.chat/s/freeplane
The planned event duration is 2 hours.

[Download calendar file](https://github.com/freeplane/freeplane/files/8597030/fpconf-2.ics.zip)

There is no registration for the event.
However, let us know by writing a comment if you going to join us.

See you!
Dimitry

**Comment by dpolivaev** [comment]
Calendar file update with the correct times and time zones (already fixed in the main post):
[Download calendar file](https://github.com/freeplane/freeplane/files/8597030/fpconf-2.ics.zip)

**Reply by macmarrum** [reply]
I'll be there

**Comment by ofossum** [comment]
Hi!

I bookmarked the date and would like to take part.

In case we need some nice tools for collaboration, i'd like to recommend some FOSS GDPR-conform tools for the session we could use:

# Brainstorming 

- __Mindwendel__ [1]
  * https://www.mindwendel.com/
  * useful for: generating ideas and vote on them
  * can be shared via link
  * people can add ideas and vote
  * contenct can be exported as CSV or HTML (summary of votes; can the be copied to CodiMD (see below))

# Notes

- **CodiMD** [2]
  * https://hackmd.okfn.de/
  * useful for: protocol and shared notes in Markdown-format
  * on the main screen, one can change the language to englich on the very
    bottom of the page
  * click `new guest note` to create a pad
  * copy the link to invite persons
  * content can be exported as Markdown, PDF and others

# Sketching

- **Excalidraw** [3]
  * https://draw.kits.blog/
  * useful for: make sketching and diagrams together
  * you can switch to english language on the right bottom of the page
  * share the session on the upper left corner (icon with people)
  * content can be exported as image

[1] : hosted in Berlin, Germany by B310
[2]: hosted by the Open Knowledge Foundation, Germay
[3]:  hosted in Germany, Lower Saxony by the Institute for Quality-Assurance in Schools

**Comment by sifran-github** [comment]
I will join :+1:

**Comment by fnatter** [comment]
I will take part.

**Comment by nnako** [comment]
I'll join.

by the way, @ofossum , nice tools. Thanks.

**Comment by EdoFro** [comment]
I will join too.

**Comment by ldmpub** [comment]
Hi all,

I would be pleased to join if there is still a seat left.

**Reply by dpolivaev** [reply]
Sure

**Comment by vboerchers** [comment]
I'll be there too

**Comment by rexelbartolome** [comment]
I'd love to join. However, there's been a lot going on over at my place so I can't really guarantee I'll be available after the next two weeks :(  I'll try to report back to this thread with some updates though :)

**Comment by HaaiHenkie** [comment]
I will be there!

**Comment by euu2021** [comment]
I will join, too

**Comment by EdoFro** [comment]
> Your curiosity sets up the agenda. You can ask any questions or present any idea or experience in a short lightning talk. The Lean Coffee meeting format allows us to be flexible and to talk about everything that most interests you as long as needed.
> 
> A very quick video introduction to Lean Coffee is available at https://www.youtube.com/watch?v=4TOOdAXwcrk .

The agenda will be set in that moment?
Are there some topics that we know it might be proposed for discussion?

I ask only because english is not my mother tongue and I need some time to search some words and think about how I can express something I want to say.

any ideas?

**Reply by ofossum** [reply]
I opened a new Mindwendel-Brainstorming for the Community-Meeting:
http://www.mindwendel.com/brainstormings/472c625a-d4ef-4de0-8aac-559f4e63803b

We can export the topics to a shared pad afterwards.
Otherwise we could as well just write a list into the shared pad on the fly.

**Comment by 1-DimensionalBox** [comment]
I will join as well!

**Comment by 1-DimensionalBox** [comment]
Thank you for yesterday's community conference! You can now find the meeting minutes as a mindmap:
  
  - [Here, packed as a .zip](https://github.com/freeplane/freeplane/files/8792818/Freeplane.online.community.conference.28.May.2022.-.minutes.mm.zip)
  -  [Here, exported to .pdf](https://github.com/freeplane/freeplane/files/8792820/Freeplane.online.community.conference.28.May.2022.-.minutes.mm.pdf)
  
  Furthermore, all questions from the LeanCoffee format have been uploaded [here, as a screenshot](https://user-images.githubusercontent.com/81996588/170856281-117a3532-534c-46c8-a695-add64beca236.png).

**Reply by 1-DimensionalBox** [reply]
And below you can find a version in MarkDown. Not too pretty, but the information is in there:

### Freeplane (online) community conference, 28 May 2022

**mobile phone application**

1-DimensionalBox: example: https://github.com/bekoeppel/droidplane

dpolivaev: "FP as a project has no resources for developing a mobile phone app."

sifran & Nnako (Nnamdi): both indicated that they would look into developing a phone app

Nnako: "I would prefer to use a modern framework"


**documentation: organization**

1-DimensionalBox: "What is current organization of the documentation? And how should it be organized?" "It seems there are multiple layers of organization. Which layers are necessary?"

dpolivaev: "People willing to contribute to documentation can decide how it should be organized." "I will open a separate Github discussion forum for the documentation group."

**documentation: migrate old MediaWiki**

macmarrum: "We're still in progress with migrating the last information from the old MediaWiki to Docsify. But when we're done, we plan to remove it."

dpolivaev: "There is consensus already, so go ahead."

**collaboration**

nnako: "I have written a design that allows for FP collaboration"

works by locking subtrees of the map and syncing (through cloud storage services for example)

also includes a possibility for versioning through Git

dpolivaev: "it might be better to focus on the collaboration feature"

**documentation: scripting**

sifran: "the documentation for scripting is fragmented and incomplete" "I would like to help centralize this, but do not know where to start"

dpolivaev: "Excuse me; poorly supported"

EdoFro: "I am currently developing add-ons which could be used to learn how to script in FP."

**documentation: standards for screenshots**

ofossum: "This could give a nicer style."

Rexel: uses xshare. only for Windows, but similar cross-platform tools exist

dpolivaev: "An upside to not using a standardized style is that this shows the diversity of FP."

**HaalHenkie: why not Java 17?**

dpolivaev: "Because it has made some major changes, and I was worried that these might cause trouble for users."

**rexelbartolome: use Github releases instead of Sourceforge?**

dpolivaev: "I don't see any real advantages. It would not necessarily be more straightforward."

**ofossum: more frequent meeting with documentation team?**

this could be organized through rexelbartolome's Discord channel

**Next meeting: 23 July, 14:00 CET**

**documentation: extra discussion**

1-DimensionalBox: "we should limit the information on the landing page of the wiki"

sifan: "The old wiki should be copied and abandoned as soon as possible, otherwise a lot of things will remain unclear."
macmarrum: "We have 90% of the work done. It won't take much more time."

1-DimensionalBox: "Regarding the interaction between GH discussions / issues and the wiki, could we label informative threads as 'include in docs', to keep track of these and make things easier for the Docs team?
EdoFro: "I know this functionality exists"
dpolivaev: "I'm thinking about who should have the responsibility for labeling and such."

1-DimensionalBox: "we should consider what documentation we want to include in Freeplane itself" "on the other hand, documentation online is more easily findable using search engines people know"
dpolivaev: "There's no problem with some redundancy from manuals." "We could convert documentation files to mindmaps."

**Reply by 1-DimensionalBox** [reply]
@HaaiHenkie, as you can read above, your question about Java was also discussed.

**Reply by dpolivaev** [reply]
The mentioned open source screen shot tool available for all platforms was https://github.com/weiameili/Pensela

**Reply by dpolivaev** [reply]
It looks like it does the wrong job: instead of editing screenshots, it allows to draw directly on the desktop before the screenshot is taken. For our use case it seems to be not as helpful as simply editing the picture afterwards.

**Reply by ldmpub** [reply]
I gave Pensela a try and my conclusion is that Pensela is unable to save any screenshot under Windows (10 / 64 bits), see:
https://github.com/weiameili/Pensela/issues/21#issuecomment-1140418938

**Reply by macmarrum** [reply]
I propose https://flameshot.org/. It works well on Linux. If someone could try it on the other platforms to confirm, it would be great.

**Reply by rexelbartolome** [reply]
Yes, Pensela also isn't that versatile in terms of changing font sizes or font styles if I remember correctly. 
> I propose https://flameshot.org/. It works well on Linux. If someone could try it on the other platforms to confirm, it would be great.

I'll go test it out on Windows :)

**Reply by rexelbartolome** [reply]
Flameshot works well in Windows, much more feature rich than Pensela but slightly less than ShareX. Mainly that text can't have an outline or it's own opaque text box, so putting down text requires you to have a good background to pair it with to make it readable. Also has a bug with the `Active Tool Size` slider, it doesn't work if you slide it manually, but scrolling the mouse wheel works to change the size of the active tool.

It also provides an option to manually input a hexcode for the colors of the text and arrows, might be useful for making the colors used in the screenshots to be consistent. 

I believe we can use Flameshot then :D maybe someone on MacOS can test it too so we can finally decide.
